networking/common/include.sh

```diff
@@ -5,7 +5,6 @@ NETWORK_COMMONLIB_DIR=$(dirname $(readlink -f $BASH_SOURCE))
 networkLib=$NETWORK_COMMONLIB_DIR
 
 # include beaker default environmnet
-. /usr/bin/rhts_environment.sh
 . /usr/share/beakerlib/beakerlib.sh || . /usr/lib/beakerlib/beakerlib.sh
```

Comment:
Shouldn't this be `. /usr/share/beakerlib/beakerlib.sh || exit 1`

Reply:
will update with exit 1 on next push including sync-up with master